### DISCUSSION TITLE
Open GPX Tracker V1.8.0 - Release to App Store (Rejected -> Approved)

### DISCUSSION THREAD

**Discussion by merlos** [merlos/iOS-Open-GPX-Tracker#224] [Announcements]
Dear Open GPX Tracker community,

Some updates. As you may know, it has been a while since the last version was released in the App Store, however, there have been many contributions from you and there were many features added that were not published.

A couple of weeks ago (20th May) the [latest version of the code (1.8.0)](https://github.com/merlos/iOS-Open-GPX-Tracker/releases/tag/releases%2Fv1.8.0) was sent to Apple to be released to the App Store. However, it has been rejected. 

Below you have the weird reason provided. 
 
----------

_Hello,_

_Thank you for submitting your app for review._

_We need additional time to evaluate your submission and Apple Developer Program account. Your submission status will appear as "Rejected" in App Store Connect while we investigate. However, we do not require a revised binary or additional information from you at this time._ 

_If we notice any issues that require your attention, we will let you know via App Store Connect. If we find no issues with your submission or account, the submission will be approved._

_If you have not heard back from us after 7 business days from the date you received this message and would like to inquire about the status of our review, you may file a request via the [Apple Developer Contact Us](https://developer.apple.com/contact/app-store/?topic=status) page._

_Best regards,_

_App Store Review_

----

I have tried to contact Apple a couple of times since then but no answer for now. 
I'll keep you posted. 

Best, Juan.

**Comment by vincentneo** [comment]
That's such a weird rejection. Have you tried resubmitting the same thing with a different build number?

**Reply by merlos** [reply]
No. On the message they say _"...we do not require a revised binary or additional information from you at this time."_

Eventually if they do not answer, I may try it...

**Reply by JohnAvatar** [reply]
I've experienced this many times with apple!!

 Have you tried submitting to the Apple review board (I forgot exact name).  Apple has a separate team of people who are not the App Reviewers that you can ask for reconsideration and sometimes that works!  It's a link at the very bottom/last of their App Review message.

If you need, I can upload an image of that link.  It's not obvious!  It was a while ago when i last do it and I'm on my iPhone, need to be on my Mac!:)

JOHNAVATAR.com

**Comment by EdCafferata** [comment]
it also can help if you put some info about GDPR (AVG) in the about.html. :-) there was a lot change about security rules.
sample:    
<p>NO ACCOUNTS ON THIS APP.</p>
  <p><a href="https://ec.europa.eu/info/law/law-topic/data-protection_nl">GDPR (AVG)</a>
<p>ALL DATA WILL BE GONE AFTER DELETE APP</p>
  <p><a href="https://support.apple.com/en-gb/guide/iphone/iph248b543ca/ios">DELETE App</a>

**Reply by merlos** [reply]
Good news everyone!

Apple has published the 1.8.0 version. 

At the end, I didn't have to do any additional action. Yesterday they changed the status to "in review" and then they released it.

Thank you for your support and for sharing your experiences.

Best, Juan!

**Comment by EdCafferata** [comment]
Hi Juan,
Very Good ;-)
Kind regards, Ed

The IT Crowd

Op 7 jun. 2022 om 10:13 heeft merlos ***@***.***> het volgende geschreven:

﻿

Good news everyone!

Apple has published the 1.8.0 version.

At the end, I didn't have to do any additional action. Yesterday they changed the status to "in review" and then they released it.

Thank you for your support and for sharing your experiences.

Best, Juan!

**Comment by seamusdemora** [comment]
That's pretty lame of Apple. Maybe the courts or governments will finally take action against their monopolistic practices. Frankly, I've just about had it with Apple products: poor documentation, mac os still distributes 20-year-old CLI tools 0n a $4K MBP, arrogant clerks in their stores, weak support even after buying Apple Care, high prices: $1,000+ for a cell phone?!?!, Siri spying, etc, etc, etc.
